user/forms.py

```diff
@@ -6,36 +9,57 @@
 
 class CustomUserCreationForm(UserCreationForm):
     """
-    This class represents a form the create a new user.
+    A form to create a new user.
     """
     class Meta(UserCreationForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
+        """
         model = User
         fields = UserCreationForm.Meta.fields + ('email', 'first_name', 'last_name', 'organisation', 'type_of_use',)
 
 
 class CustomUserChangeForm(UserChangeForm):
     """
-    This class represents a form to change attributes of a user. The Form is used in the admin area.
+    A form to change attributes of a user. This form is only used in the admin area.
     """
     class Meta(UserChangeForm.Meta):
+        """
+        Metadata for the model User. Selects which fields should be editable in this form.
+        """
         model = User
         fields = '__all__'
 
 
 class CustomUserEditForm(UserChangeForm):
     """
-    This class represents a form to change attributes of a user by himself.
+    A form to change attributes of a user by himself.
+
+    Attributes
+    ----------
+    password : str
+        None - to erase the password in the form.
```

Comment:
None would be a special case, correct? Then please write it that way. Maybe something like "Password, which must be at least X characters long. In case it is set to None the password ...".